lsp-mode.el

```diff
@@ -9934,6 +9957,43 @@ string."
     (setf window-scroll-functions
           (delete #'lsp--update-inlay-hints-scroll-function window-scroll-functions)))))
 
+
+;; Inline Completions
```

Comment:
nit: If possible we should move this to `lsp-inline-completion.el` instead.
An auto hook can be added to `lsp-inline-completion` to autoload this feature if it's on.
Example:
``` elisp
;;;###autoload
(add-hook 'lsp-configure-hook (lambda ()
                                (when (and lsp-auto-configure
                                           lsp-completion-enable)
                                  (lsp-completion--enable))))
```

Reply:
Oh, that's a neat trick!